vald/templates/discoverer/clusterrole.yaml

```diff
@@ -43,4 +43,5 @@ rules:
   - /metrics
   verbs:
   - get
+  - list
```

Comment:
![badge](https://img.shields.io/badge/review-must-red.svg?style=flat-square)
this line doesn't needed.
However, it is needed to add this role:
```yaml
- apiGroups:
  - "metrics.k8s.io"
  resources:
  - nodes
  - pods
  verbs:
  - get
  - list
```

Reply:
fixed